pkg/osquery/runtime/osqueryinstance.go

```diff
@@ -249,6 +249,12 @@ func (i *OsqueryInstance) Launch() error {
 		return fmt.Errorf("could not calculate osquery file paths: %w", err)
 	}
 
+	// Make sure no lock files exist that could prevent osquery from starting up
+	if err := i.cleanUpOldDatabaseLock(ctx, paths); err != nil {
```

Comment:
probably want to do this in a loop, windows can be so finicky about deleting files in use

Reply:
Oh yes, thank you! I'll move this to a backoff/retry